tests/test_export_client.py

```diff
@@ -0,0 +1,126 @@
+import json
+from importlib import resources
+
+import pytest
+from httpx import URL, HTTPStatusError
+
+from aaq_sync.data_export_client import ExportClient
+from aaq_sync.data_models import FAQModel
+
+from .fake_data_export import FakeDataExport
+
+
+@pytest.fixture()
+def fake_data_export(httpx_mock):
+    return FakeDataExport(URL("https://127.0.0.100:1234/"), httpx_mock)
```

Comment:
Noted usage of httpx_mock, would be great to use this in the rdw-ingestion-tools package. Adding ticket to my backlog.

Reply:
I don't know if it's worth splitting the export client into a separate library (without the data model stuff) that both this and your ingestion tools can use, but if it isn't we can put a modified copy over there.

If we do the separate library thing, we should probably include `FakeDataExport` to make testing the things that use it easier.